iset.mm

```diff
@@ -64121,6 +64129,112 @@ all reals (greatest real number) for which all positive reals are greater.
+        -> ( E. x e. y ( f ` x ) = (/) \/ A. x e. y ( f ` x ) = 1o ) ) } $.
+  $}
+
+  ${
+    $d A f x y $.
+    $( Expansion of ~ df-omni in terms of classes.  (Contributed by Jim
+       Kingdon, 28-Jun-2022.) $)
+    dfomnicl $p |- ( A e. V -> ( A e. Omni <-> A. f ( f : A --> 2o
+        -> ( E. x e. A ( f ` x ) = (/) \/ A. x e. A ( f ` x ) = 1o ) ) ) ) $=
+      ( vy cv c2o wf cfv c0 wceq wrex c1o wral wo wi wal comni feq2 rexeq raleq
+      orbi12d imbi12d albidv df-omni elab2g ) EFZGCFZHZAFUHIZJKZAUGLZUJMKZAUGNZ
+      OZPZCQBGUHHZUKABLZUMABNZOZPZCQEBRDUGBKZUPVACVBUIUQUOUTUGBGUHSVBULURUNUSUK
+      AUGBTUMAUGBUAUBUCUDAECUEUF $.
+  $}
+
+  ${
+    $d A w y z $.  $d f g x y $.  $d g u x y z $.
+    $( A finite set is omniscient.  Remark right after Definition 3.1 of
+       [Pierik], p. 14.  (Contributed by Jim Kingdon, 28-Jun-2022.) $)
+    finomni $p |- ( A e. Fin -> A e. Omni ) $=
+      ( vz vf vx vg vu cv comni wcel c0 c2o cfv wceq c1o wral wo wi cvv eqeq1d
+      wa vw vy csn cun eleq1 wf wrex wal wb 0ex dfomnicl ax-mp ral0 olci mpgbir
+      a1i cres elun1 ad2antlr fvres simpr eqtr3d fveq2 rspcev syl2anc rexlimdva
+      cfn ex vsnid elun2 sylan a1d cbvralv ralbiia bitri sylib simplr vex ralsn
+      orcd sylibr ralun olcd cpr ffvelrnd df2o3 syl6eleq mpjaodan biimpi adantl
+      elpri syl resex feq1 fveq1 rexbidv ralbidv orbi12d imbi12d spcv wss ssun1
+      fssres mpan2 impel mpjaod alrimiv snex unex findcard2 ) UAGZHIJHIZUBGZHIZ
+      XMBGZUCZUDZHIZAHIUAUBBAXKJHUEXKXMHUEXKXQHUEXKAHUEXLJKCGZUFZDGZXSLZJMZDJUG
+      ZYBNMZDJOZPZQZCJRIXLYHCUHUIUJDJCRUKULYGXTYFYDYEDUMUNUPUOXMVGIZXNXRYIXNTZX
+      QKEGZUFZFGZYKLZJMZFXQUGZYNNMZFXQOZPZQZEUHZXRYJYTEYJYLYSYJYLTZYAYKXMUQZLZJ
+      MZDXMUGZYSUUDNMZDXMOZUUBUUEYSDXMUUBYAXMIZTZUUEYSUUJUUETZYPYRUUKYAXQIZYAYK
+      LZJMZYPUUIUULUUBUUEYAXMXPURUSUUKUUDUUMJUUIUUDUUMMUUBUUEYAXMYKUTUSUUJUUEVA
+      VBYOUUNFYAXQYMYAMYNUUMJYMYAYKVCSVDVEVTVHVFUUBXOYKLZJMZUUHYSQUUONMZUUBUUPT
+      ZYSUUHUURYPYRUUBXOXQIZUUPYPUUSUUBXOXPIUUSBVIXOXPXMVJULUPZYOUUPFXOXQYMXOMZ
+      YNUUOJYMXOYKVCZSVDVKVTVLUUBUUQTZUUHYSUVCUUHTZYRYPUVDYQFXMOZYQFXPOZYRUVDUU
+      HUVEUVCUUHVAUUHYMUUCLZNMZFXMOUVEUUGUVHDFXMYAYMMUUDUVGNYAYMUUCVCSVMUVHYQFX
+      MYMXMIUVGYNNYMXMYKUTSVNVOVPUVDUUQUVFUUBUUQUUHVQYQUUQFXOBVRZUVAYNUUONUVBSV
+      SWAYQFXMXPWBVEWCVHUUBUUOJNWDZIUUPUUQPUUBUUOKUVJUUBXQKXOYKYJYLVAUUTWEWFWGU
+      UOJNWKWLWHYJXMKUUCUFZUUFUUHPZYLYJXMKXSUFZYCDXMUGZYEDXMOZPZQZCUHZUVKUVLQZX
+      NUVRYIXNUVRXMRIXNUVRUIUBVRZDXMCRUKULWIWJUVQUVSCUUCYKXMEVRWMXSUUCMZUVMUVKU
+      VPUVLXMKXSUUCWNUWAUVNUUFUVOUUHUWAYCUUEDXMUWAYBUUDJYAXSUUCWOZSWPUWAYEUUGDX
+      MUWAYBUUDNUWBSWQWRWSWTWLYLXMXQXAUVKXMXPXBXQKXMYKXCXDXEXFVHXGXQRIXRUUAUIXM
+      XPUVTXOUVIXHXIFXQERUKULWAVHXJ $.
+  $}
+
+  ${
+    $d f x y $.  $d ph x y z $.
+    omniexmid.1 $e |- x e. Omni $.
+    $( If all sets are omniscient, excluded middle follows.  (Contributed by
+       Jim Kingdon, 29-Jun-2022.) $)
+    omniexmid $p |- ( ph \/ -. ph ) $=
```

Comment:
Why not use `DECID ph` ?

Reply:
I guess there is a bit of a stylistic question, at least in my mind, about whether to use definitions like `DECID` everywhere we can, or just where it is hard not to. This thing about `ph \/ -. ph` as the conclusion of an *exmid theorem is what we have been doing, for whatever it is worth.

It is a moot point, however, if we use `EXMID` instead (as this pull request now does). So maybe there is no need to worry about it much any more.